data/ch/skill.yml

```diff
@@ -0,0 +1,26 @@
+################################# Skill ##################################
+skill:
+  enable : true
+  title : Our Skills
+  subtitle : We’ve skilled in wide range of web and Other digital market tools.
+  content : Lorem, ipsum dolor sit amet consectetur adipisicing elit. Perspiciatis magni explicabo cum aperiam recusandae sunt accusamus totam. Quidem quos fugiat sapiente numquam accusamus quas hic, itaque in libero reiciendis tempora!
+  image : images/about/company-growth.webp
+  skill_item :
+    # skill item loop
+    - name :  Business Development
+      percent : 90%
+      
+    # skill item loop
+    - name :  Analysis
+      percent : 70%
+      
+    # skill item loop
+    - name :  Design
+      percent : 80%
+      
+    - name :  IOS Development
+      percent : 95%
+      
+    # skill item loop
+    - name :  Android Development
+      percent : 75%
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)